src/db/entity/sms.otp.ts

```diff
@@ -10,22 +10,10 @@ export class SmsOtp {
     id: number;
 
     /**
-     * The agent id of the entity this record hold the phone number for
+     * The did of the entity this record hold the phone number for
      */
```

Comment:
Im a little bit concerned about terminology.   Prior to the change we were pretty explicit that the id agent id.   now we just call this a did and we have lost context as whom the did applies.  (in indy dids used a lot more).  Its a minor thing but I feel like we need to keep some context in the description.

Reply:
I agree, it is a bit concerning. There were also several spots in the code where Jacob had left notes about renaming did to agent id. When I asked him about it, he said to just use did. The nice thing is that it's actually quite easy to change within the codebase and it would be a very simple migration now (I think with synchronization it might have actually dropped the column instead of renaming it). I personally don't know enough to have an opinion about which one is better. Do you think we should go with agent id everywhere instead? And should we change it in other code bases? e.g. I think IdentityService uses did and probably some others doo too.

Reply:
theres something to be said for being consistent across the code bases.   which we are probably not doing well.   I wouldn't let this hold up the PR though.  just something to think about.